system-test/spanner.ts

```diff
@@ -3457,7 +3458,7 @@ describe('Spanner', () => {
         const query = 'SELECT * FROM TxnTable';
 
         const options = {
-          minReadTimestamp: new Date(),
+          minReadTimestamp: new PreciseDate(),
```

Comment:
I couldn't really tell from the [docs](https://github.com/googleapis/nodejs-precise-date) what PreciseDate() defaults to for the empty ctor. Is it some value or is it just an empty object?

Reply:
PreciseDate() will call ctor for super() i.e., Date() which returns Now() timestamp. TimestampBounds object however expects an object of PreciseDate and will convert it correctly to a string representation.

Reply:
https://screenshot.googleplex.com/X54iBrh6WFQ

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

Reply:
Ah got it, thanks for the pointer @agasheesh.